src/components/signup/OrgLookup.tsx

```diff
@@ -129,19 +127,15 @@ const OrgLookup = ({
   setSelectedOrganization,
   selectedOrganization,
 }: props) => {
-  /**
-   * List of avialable organizations
-   */
   const [organizations, setOrganizations] = useState([]);
-  /**
-   * Form disabled or active flag.
-   * set to "false" when an organization is selected
-   */
-  const [disableSubmit, setdisableSubmit] = useState<boolean>(true);
+  //Form disabled or active flag, set to "false" when an organization is selected
+  const [disableSubmit, setDisableSubmit] = useState<boolean>(true);
   const [askConfirmation, setAskConfirmation] = useState<boolean>(false);
+  const [isOpen, setIsOpen] = useState(false);
+  const [searchTerm, setSearchTerm] = useState('');
 
+  //List of avialable organizations
```

Comment:
It's minor but do you mind adding a space before List and before Form after the `//`?

Reply:
Need review for this PR too, not sure how it got closed